hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncNonRootRegionLocator.java

```diff
@@ -667,4 +689,14 @@ int getNumberOfCachedRegionLocations(TableName tableName) {
     }
     return tableCache.cache.values().stream().mapToInt(RegionLocations::numNonNullElements).sum();
   }
+
+  private static KeyValue.KVComparator getComparator(TableName tableName) {
```

Comment:
Is this the right place for this method?

Reply:
Ah no this is an artifact of PoC just to get things working. I will consolidate this into CellComparator as well as remove usage of KVComparator.